README.md

```diff
@@ -16,22 +16,30 @@ ActiveRecord::DynamicTimeout::Initializer.initialize!
 ```
 
 ## Usage
-    
+To use this gem, you can set a timeout for a block of code that runs queries using ActiveRecord.
+Within the block, if a query takes longer than the timeout value (in seconds), an `ActiveRecord::QueryAborted` error (or a subclass of it) will be raised.
+
+#### Example
 ```ruby
 class MyModel < ActiveRecord::Base
   # Model code...
 end
 
-MyModel.with_timeout(5) do
-  MyModel.find(1)
+# Set a timeout for all queries run within the block
+MyModel.with_timeout(5.seconds) do
+  MyModel.all # A long running query that takes over 5 seconds
 end
+# => Raises ActiveRecord::QueryAborted error (or a subclass of it) after 5 seconds.
+
+MyModel.all # A long query that takes over 5 seconds
+# => ActiveRecord::Relation<MyModel>...
```

Comment:
Thanks for adding these examples.

What do you think about swapping the order? That's usually the way I'd expect to see them, I think:

- before (not using this gem)
- after (using this gem)